vllm/model_executor/layers/fused_moe/fused_marlin_moe.py

```diff
@@ -136,7 +147,7 @@ def _fused_marlin_moe(
         size_n=2 * N,
         size_k=K,
         is_k_full=is_k_full,
-        use_atomic_add=use_atomic_add,
+        use_atomic_add=False,
```

Comment:
What happened to atomic add support?

Reply:
`atomic_add` only provides benefits when both `m_tiles` and `n_tiles` are extremely small.
In the MoE scenario, `m_tiles` will at least be equal to `top_k`, so in most cases, `atomic_add` does not offer any performance improvement (and even if it does, the gain is minimal).
Therefore, I think it’s unnecessary to compile an additional version for sm90 just for these rare cases, so I’ve completely disabled this logic.